templates/show-task-table.html

```diff
@@ -32,13 +32,15 @@
                 <tbody>
                 {% for task in tasks %}
                   <tr>
-                    <td>{{task.article_name}} {% if logged_in_user and task.status == "open" or logged_in_user and task.status == "claimed" and task.claimed_by == logged_in_user %} <i class="fa fa-pencil-square-o" onclick="taskForm({{forloop.counter0}})"></i> {% endif %} </td>
-                    <td>{{task.description}}</td>
-                    <td>{{task.created_by}}</td>
-                    <td>{{task.status}}</td>
-                    <td>{{task.progress}}</td>
-                    <td>{{task.effort}}</td>
-                    <td>{{task.claimed_by}}</td>
+                    <td><a href="https://en.wikipedia.org/wiki/{{ task.article_name|urlencode }}">{{ task.article_name }}</a>
```

Comment:
Are you sure it's `urlencode` that we want here? That turns spaces into +, for instance:

```python
>>> import urllib.parse
>>> urllib.parse.urlencode({'q': 'a b c'})
'q=a+b+c'
```

Which wouldn't work on Wikipedia: https://en.wikipedia.org/wiki/Patrice+Lumumba

IIRC for Citation Hunt we just replace spaces with underscores in the article title, and that's probably enough.

Reply:
Done!